### PR TITLE
[ONNX Import] add method Node::get_attribute_as_constant()

### PR DESCRIPTION
**Pull request by artkuli**
### Tickets:
 - *53284*

### CODE REVIEW

File: src/frontends/onnx/frontend/src/core/node.cpp [hunk 1]
```diff
@@ -203,6 +209,35 @@ const std::string& Node::Impl::description() const {
     return m_description;
 }
 
+template <typename T>
+std::shared_ptr<ov::op::v0::Constant> Node::Impl::get_attribute_as_constant(const std::string& name) const {
+    const auto value = get_attribute_value<T>(name);
+    const element::Type type = element::from<T>();
+    return ov::op::v0::Constant::create(type, Shape{}, std::vector<T>{value});
```
Comment (mateusztabaka) [inline src/frontends/onnx/frontend/src/core/node.cpp:216]: you can consider using this constructor https://github.com/openvinotoolkit/openvino/blob/master/src/core/include/openvino/op/constant.hpp#L69

Comment (artkuli) [reply src/frontends/onnx/frontend/src/core/node.cpp]: Good point!  😊  Done

**Review by tomdol — APPROVED** [review]

**Review by mbencer — APPROVED** [review]

**Review by mateusztabaka — APPROVED** [review]

**Review by mbencer — APPROVED** [review]

File: src/frontends/onnx/frontend/src/op/gemm.cpp [hunk 2]
```diff
@@ -30,10 +30,7 @@ OutputVector gemm(const Node& node) {
     }
 
     const auto alpha = node.get_attribute_value<float>("alpha", 1);
-    const auto beta = node.get_attribute_value<float>("beta", 1);
-
-    const auto alpha_node = default_opset::Constant::create(input_b.get_element_type(), Shape{}, {alpha});
-    const auto beta_node = default_opset::Constant::create(input_c.get_element_type(), Shape{}, {beta});
+    const auto beta_node = node.get_attribute_as_constant<float>("beta", 1, input_c.get_element_type());
```
Comment (mateusztabaka) [inline src/frontends/onnx/frontend/src/op/gemm.cpp:33]: I think you can do the same for 'beta' from set_6::gemm

Comment (artkuli) [reply src/frontends/onnx/frontend/src/op/gemm.cpp:33]: Done

**Review by mateusztabaka — APPROVED** [review]

**Review by tomdol — APPROVED** [review]

**Pull request merged**